core/arch/arm/plat-imx/conf.mk

```diff
@@ -460,5 +460,30 @@ CFG_IMX_CAAM ?= y
 endif
 endif
 
+# This is only here to force an informative print showing that the device runs
```

Comment:
I would wait for NXP to publish the disclaimer before adding this chunk of text. Some information added in this file will also be specified in the disclaimer NXP is currently working on.

I would also personally prefer to have explanatory text in the documentation rather than in the source code.
Let me know if waiting for the disclaimer before merging is okay for you.